src/application-insights/azext_applicationinsights/_params.py

```diff
@@ -37,7 +38,7 @@ def load_arguments(self, _):
         c.argument('resource_group_name')
 
     with self.argument_context('monitor app-insights query') as c:
-        c.argument('application', options_list=['--app', '--apps', '-a'], nargs='+', id_part='name', help='GUID, app name, or fully-qualified Azure resource name of Application Insights component. The application GUID may be acquired from the API Access menu item on any Application Insights resource in the Azure portal. If using an application name, please specify resource group.')
```

Comment:
This should just be `--apps`. Argpase will accept `--app` anyways based on how it parses options.